pint.json

```diff
@@ -25,6 +25,7 @@
         "declare_strict_types": true,
         "single_space_around_construct": false,
         "statement_indentation": false,
+        "line_ending": false,
```

Comment:
Don't think we should drop this rule, as it keeps consistent line-endings within this package